doc/api/toolkits/mplot3d/view_angles.rst

```diff
@@ -38,3 +38,156 @@ further documented in the `.mplot3d.axes3d.Axes3D.view_init` API.
 
 .. plot:: gallery/mplot3d/view_planes_3d.py
    :align: center
+
+
+.. _toolkit_mouse-rotation:
+
+Rotation with mouse
+===================
+
+3D plots can be reoriented by dragging the mouse.
+There are various ways to accomplish this; the style of mouse rotation
+can be specified by setting ``rcParams.axes3d.mouserotationstyle``, see
+:doc:`/users/explain/customizing`.
```

Comment:
I like your replacement :rc:`axes3d.mouserotationstyle`; however, it links to the middle of the :doc:`/users/explain/customizing` page, not to the top, so I wonder, should we not keep the last part: see [Customizing Matplotlib with style sheets and rcParams](https://output.circle-artifacts.com/output/job/fb4784cd-5234-45af-a614-b0986dbc94eb/artifacts/0/doc/build/html/users/explain/customizing.html) ?

Reply:
Linking using :rc: is logically correct. We will have better rcParams docs in the future so that this link will directly go to the definition of the parameter.

Reply:
> Linking using :rc: is logically correct. 

Of course, I have no issue with that, that's definitely an improvement, consider it done.

> We will have better rcParams docs in the future so that this link will directly go to the definition of the parameter.

That would be a nice improvement too - but the :rc: would still just point to the definition of the parameter itself, rather than to the instructions on Customizing with style sheets or rcParams, I thought that's what's most relevant - but if you think that's just too much duplication, please say so.